peekingduck/runner.py

```diff
@@ -29,9 +29,8 @@ class Runner():
     """
     The runner class for creation of pipeline using declared/given nodes.
 
-    The runner node situationally, depending on different inputs, use the
-    provided configurations to setup a node pipeline whih is used to run single
-    or continuous inference.
+    The runner node uses the provided configurations to setup a node pipeline
+    which is used to run single or continuous inference.
```

Comment:
Dont think the runner can be considered a node right? 
```suggestion
    The runner class uses the provided configurations to setup a node pipeline
    which is used to run inference.
```